content/4_Gold/Tree_Euler.mdx

```diff
@@ -542,6 +564,86 @@ public class Main {
 
 </JavaSection>
 
+<PySection>
+
+```py
+import sys
+
+NMAX = 200005
+n, m = map(int, input().split())
```

Comment:
why read as `m`? Isn't it 'q' in the official statement?

Reply:
I changed $m$ to $q$